include/integratorxx/quadratures/gausscheby1.hpp

```diff
@@ -64,16 +65,18 @@ struct quadrature_traits<GaussChebyshev1<PointType, WeightType>> {
     weight_container weights(npts);
 
     const weight_type pi_ov_npts = M_PI / npts;
-    const weight_type two_npts_x_pi = 2 * npts * M_PI;
+    const weight_type pi_ov_2npts = pi_ov_npts / 2;
 
     for (size_t i = 0; i < npts; ++i) {
+      const auto ti = (2.0 * (i + 1) - 1.) * pi_ov_2npts;
       // The standard nodes and weights are given by
-      points[i] = std::cos((2.0 * (i + 1) - 1.) / two_npts_x_pi);
-      weights[i] = pi_ov_npts;
+      points[i] = -std::cos(ti);
```

Comment:
Why the minus sign?!

Reply:
This is ensures that the quadrature is generated in ascending order, which is important for ensuring the radial transforms are in ascending order (which is the format needed to interop with the pruner for the spherical quadratures) - I believe Psi4 generated in descending order, potato potato.

Reply:
On second thought, this should be done via indexing ala #34, changes back for now